handbook/network-security/network-security.md

```diff
@@ -0,0 +1,225 @@
+
+### DNS Configuration
+
+SystemLink and NI Web Server do not ship with a DNS server. NI presumes you will provide a DNS server to be used in your environment. NI Web Server will attempt to provide a default for the DNS of your SystemLink server based on Windows OS settings. Other valid DNS names are listed in the **Preferred host name for generated URLs and certificates** combo box. You may also manually additional hostnames as needed. As the name of the combo box implies, this setting must match the host name in your TLS certificates. If an invalid hostname is provided data from managed targets will not be published.
+
+### CORS and Remote Connections
+
+NI recommends disabling CORS in production environments to achieve a higher degree of security. In practice, CORS may need to be enabled to facilitate workflows for users developing web applications that interact with SystemLink's API. For this scenario, NI recommends setting up a *test* SystemLink server with CORS enabled.
+
+NI Web Server Configuration exposes settings for choosing a remote connection and automatically sets Windows Firewall rules. This ensures connections may only be established by clients on your preferred network.
+
+Refer to [Choosing Remote Settings](https://www.ni.com/documentation/en/ni-web-server/latest/manual/choosing-a-remote-setting/) for details on the various CORS and remote connection settings.
+
+## Target to SystemLink communication
+
+Test Systems communicate with SystemLink over HTTP(S) and the SaltStack TCP protocol. Refer to the SystemLink manual for prerequisites and steps to add a [Linux RT]((https://www.ni.com/documentation/en/systemlink/latest/setup/setting-up-systemlink-client-linux/)) or [Windows](https://www.ni.com/documentation/en/systemlink/latest/setup/setting-up-systemlink-client-windows/) target to your SystemLink server.
+
+Data published over HTTPS includes tags, files (FIS), assets, and test results. Salt jobs and pillars communicate over the AES encrypted Salt TCP transport. Salt jobs are used for installing software and changing target configuration from SystemLink server. Salt pillars are used to transfer credentials and certificates. The certificates used on the SystemLink server and target nodes are managed by Salt and do not require administrators to externally manage these certificates. Refer to SaltStack's [documentation](https://docs.saltproject.io/en/getstarted/system/communication.html) for an overview of the Salt TCP Transport.
+
+!!! note "Managed NI LinuxRT Nodes"
+    NI recommendations changing the default username and password on the target. These credentials are required to SSH into the target. These credentials are required when a SystemLink server adds a Linux RT target to its collection of managed systems.
```

Comment:
It is not possible to change the default username on a Linux RT target. What we recommend as assigning a (strong) password to the "admin" user on the target.